auth/server/src/main/java/com/seagates3/authentication/ClientRequestParser.java

```diff
@@ -59,8 +59,9 @@ public static ClientRequestToken parse(FullHttpRequest httpRequest,
         String authorizationHeader;
         ClientRequestToken.AWSSigningVersion awsSigningVersion;
 
-        if (requestAction.equals("AuthenticateUser")
-                || requestAction.equals("AuthorizeUser")) {
+        if (requestAction.equals("AuthenticateUser") ||
```

Comment:
Will we also be having seaprate Authenticate and Authorize  requests ?

Reply:
Pure authentication is needed for s3 background services when authorization is made by s3server itself.
Pure authorization is needed for anonymous requests and copy object